root/static/search_plugins/opensearch/musicbrainz_artist.xml

```diff
@@ -6,10 +6,10 @@
   <Description>Indexed search for an artist on MusicBrainz.org</Description>
   <Url type="text/html" method="get" template="http://musicbrainz.org/search/textsearch.html?type=artist&amp;limit=25&amp;handlearguments=1&amp;query={searchTerms}"/>
   <Developer>https://metabrainz.org/contact</Developer>
-  <Image width="16" height="16">http://musicbrainz.org/static/images/entity/artist.png</Image>
+  <Image width="16" height="16">http://musicbrainz.org/static/images/entity/artist.svg</Image>
```

Comment:
Not sure if that's going to work, to be honest. Reading https://developer.mozilla.org/en-US/Add-ons/Creating_OpenSearch_plugins_for_Firefox I only see ico, jpeg, and png mentioned. Perhaps we should still keep some rendered pngs in the repository.